libr/anal/p/anal_tricore.c

```diff
@@ -0,0 +1,140 @@
+#include <string.h>
+#include <r_types.h>
+#include <r_lib.h>
+#include <r_asm.h>
+#include <r_anal.h>
+#include "../../asm/arch/tricore/gnu/tricore-opc.c"
+
+static bool set_reg_profile(RAnal *anal) {
+
+	const char *p = NULL;
+	switch (anal->bits) {
+	case 32: p =
+		"=PC	pc\n"
+		"=SP	a10\n"
+		"gpr	a0	.32	0	0\n"
```

Comment:
this loks wrong to me. .32 is 32bits, but 32 is 32 bytes... so that regprofile is taking much more memory that it should

Reply:
Check now.